test/provision/compile.sh

```diff
@@ -4,15 +4,14 @@ set -e
 CILIUM_DS_TAG="k8s-app=cilium"
 KUBE_SYSTEM_NAMESPACE="kube-system"
 KUBECTL="/usr/bin/kubectl"
+PROVISIONSRC="/tmp/provision"
+GOPATH="/home/vagrant/go"
 
-export GOPATH="/go/"
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
```

Comment:
Why you added DIR here?

Reply:
it was a copy paste from the other scripts. It doesn't hurt